integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -153,6 +153,20 @@ def _transmit_action(self, content_metadata_item_map, client_method, action_name
         for chunk in islice(chunk_items, transmission_limit):
             json_payloads = [item.channel_metadata for item in list(chunk.values())]
             serialized_chunk = self._serialize_items(json_payloads)
+            if self.enterprise_configuration.dry_run_mode_enabled:
+                encoded_serialized_payload4 = encode_binary_data_for_logging(serialized_chunk)
+                enterprise_customer_uuid = self.enterprise_configuration.enterprise_customer.uuid
+                LOGGER.info(generate_formatted_log(
+                    self.enterprise_configuration.channel_code(),
+                    enterprise_customer_uuid,
+                    None,
+                    None,
```

Comment:
another reason why you may want to log for each item here is so that you can set the `course_or_course_run_key` variable in the `generate_log_line` method - otherwise it might be difficult to trace the lines for specific content items.